docs/user-guide/settings/messages-feishu.md

```
@@ -134,7 +138,20 @@ start_bot      → 启动 feishu_listener tokio task
 - 同时按 `chat_id` 聚合到 `feishu_history_chats`
 - `message_debounce` 服务：5 分钟内同一群的高频消息做去重，避免刷屏
 
-### 5.3 API
+###5.3消息处理状态（`processed` / `failed`）
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)